### PR TITLE
fixes #8045 tests(nimbus): Add an integration test for sticky targeting

### LINKED ISSUE #8045: Integration test for Sticky Targeting
Let’s write an e2e integration test for sticky targeting

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2998)

### PR DESCRIPTION
**Pull request by jrbenny35**
Because:
* Sticky targeting is alive and well within nimbus

This Commit:
* Ensures we have proper automated testing of it.

### CODE REVIEW

File: app/experimenter/targeting/constants.py [hunk 1]
```diff
@@ -825,6 +825,17 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TEST_STICKY_TARGETING = NimbusTargetingConfig(
+    name="Test targeting",
+    slug="test_targeting",
+    description="Config for sticky targeting",
+    targeting="'sticky.targeting.test.pref'|preferenceValue",
+    desktop_telemetry="",
+    sticky_required=False,
```
Comment (eliserichards) [inline app/experimenter/targeting/constants.py:834]: Why wouldn't this be `sticky_required=True`?

**Review by eliserichards — COMMENTED** [review]
Looks good! One tiny question for you :D

File: app/experimenter/targeting/constants.py (hunk 1 above)
Comment (jrbenny35) [reply app/experimenter/targeting/constants.py:834]: So in the test I set the sticky field itself, but seeing as this is a sticky targeting pref, it should be set here too I guess. I can change it

Comment (eliserichards) [reply app/experimenter/targeting/constants.py:834]: Ok sweet. I wasn't sure if that would be a separate test or something 👍

**Pull request merged**